plugins/premium/mentions.md

```diff
@@ -240,9 +240,9 @@ tinymce.init({
 
 ### `mentions_select`
 
-This option enables an element to be presented below a mention when a user hovers over it on the page. This could include details about the user. A pre-defined template can be specified by name or a custom hover element can be configured using a callback function.
+This option enables an element to be presented below a mention when a user hovers over it on the page. This could include details about the user. A custom hover element can be created or a predefined template can be specified.
```

Comment:
"element" -> "popup"?

Also, is it always above, or can it appear below?

"on the page" -> "in the editor"? This implied it would work on the published content, which I believe people would have to set up themselves separately?

Reply:
```suggestion
This option enables a hover card to be presented when a user hovers over a mention in {{site.productname}}. This could include details about the user. A custom hover card can be defined or a predefined template can be specified.
```

Reply:
We went looking and "hover card" seems to be a common enough term - e.g. Chrome and Facebook use it, and they're similar concepts.

Reply:
Hover card sounds better, thanks for the suggested changes.